next.config.mjs

```diff
@@ -1,9 +1,12 @@
 'use strict';
 
 import * as nextConstants from './next.constants.mjs';
+import nextRewrites from './next.rewrites.mjs';
```

Comment:
what do you thinks about putting all file like this in an folder like "next-utils" ?